### ISSUE TITLE
Alternate Email Address Labeling (accessibility)

### ISSUE THREAD

**Issue by jonibar** [psu-stewardship/etda#88]
Alternate email address label should have an asterisk to indicate it is a required field.

Include instructions that alternate email cannot be a PSU email address.
